experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -1260,16 +1282,28 @@ julia> L = special_linear_lie_algebra(QQ, 3);
 
 julia> V = exterior_power(standard_module(L), 2)[1]; # some module
 
-julia> T, map = tensor_power(V, 3)
-(Tensor power module of dimension 27 over sl_3, Map: parent of tuples of type Tuple{LieAlgebraModuleElem{QQFieldElem}, LieAlgebraModuleElem{QQFieldElem}, LieAlgebraModuleElem{QQFieldElem}} -> tensor power module)
+julia> T, map = tensor_power(V, 2)
+(Tensor power module of dimension 9 over sl_3, Map: parent of tuples of type Tuple{LieAlgebraModuleElem{QQFieldElem}, LieAlgebraModuleElem{QQFieldElem}} -> tensor power module)
 
 julia> T
 Tensor power module
-  of dimension 27
-  3rd tensor power of
+  of dimension 9
+  2nd tensor power of
     2nd exterior power of
       standard module
 over special linear Lie algebra of degree 3 over QQ
+
+julia> basis(T)
+9-element Vector{LieAlgebraModuleElem{QQFieldElem}}:
+ (v_1∧v_2) ⊗ (v_1∧v_2)
+ (v_1∧v_2) ⊗ (v_1∧v_3)
+ (v_1∧v_2) ⊗ (v_2∧v_3)
+ (v_1∧v_3) ⊗ (v_1∧v_2)
+ (v_1∧v_3) ⊗ (v_1∧v_3)
+ (v_1∧v_3) ⊗ (v_2∧v_3)
+ (v_2∧v_3) ⊗ (v_1∧v_2)
+ (v_2∧v_3) ⊗ (v_1∧v_3)
+ (v_2∧v_3) ⊗ (v_2∧v_3)
```

Comment:
While adding some doctests for this and similar cases, I noticed that \otimes is used as well. Do you have a suggestion for some non-unicode way to print these? @fingolfin

Reply:
Sorry, no great solution from me. I've seen all kinds of hacks for this like `(x)`, `x`; `#`; ...   oh  and also `/\` instead of `^` for `∧`. So e.g.
- `(v_2^v_3) (x) (v_2^v_3)`
- `(v_2^v_3) x (v_2^v_3)`
- `(v_2/\v_3) (x) (v_2/\v_3)`
- `(v_2/\v_3) x (v_2/\v_3)`

Frankly none of these are great.

Reply:
I now use `(x)` for the time being.